tests/unit/test-util/test.adb.js

```diff
@@ -223,12 +223,28 @@ describe('utils/adb', () => {
         testFn: (adbUtils) => adbUtils.discoverInstalledFirefoxAPKs('device1'),
       });
 
+      sinon.assert.calledOnce(adb.fakeADBDevice.shell);
+    });
+
+    it('retrieves current user', async () => {
```

Comment:
Please add a test case with a userId other than 0, just for completeness. For example 1234.

Reply:
I propose updating this testcase to test non-zero ID, since zero ID is already covered by 'resolves the array of the installed firefox APKs' test.

Reply:
I agree with the explanation, the `resolves the array of the installed firefox APKs' does not test the `am get-current-user` result parsing.
Still we have the result parsing covered by positive and negative test already. While '0' might have not been the most representative value (updated), I'm not sure I see a reason in replicating the test to cover '0' vs '123', or any other numeric ID for that matter. That code has 100% coverage already.